src/components/ReadMore/index.js

```diff
@@ -2,28 +2,20 @@ import React from 'react';
 import {View, Text, StyleSheet, TouchableOpacity} from 'react-native';
 import {COLORS} from '../../utils/theme';
 import useReadmore from './hooks/useReadmore';
+import {fonts, normalizeFontSize} from '../../utils/fonts';
 
 const styles = StyleSheet.create({
+  text: {
+    fontFamily: fonts.inter[400],
+    fontSize: normalizeFontSize(14),
+    color: COLORS.grey410
+  },
   moreText: {
     color: COLORS.bondi_blue
   }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code introduces a `text` style in the StyleSheet, which includes fontFamily, fontSize, and color. This is a good practice as it ensures consistency across the application and makes it easier to manage changes. However, it seems that the `containerStyle` and `textStyle` props have been removed from the `ReadMore` component. These props allow for more flexible styling of the component when it's used in different contexts. If there's no specific reason for removing these props, I would suggest keeping them.

```diff
- const ReadMore = ({text, onPress}) => {
+ const ReadMore = ({text, onPress, containerStyle, textStyle}) => {
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->